templates/apis/crd.go.tpl

```diff
@@ -16,6 +16,7 @@ import (
 // {{ .CRD.Kind }}Spec defines the desired state of {{ .CRD.Kind }}
 type {{ .CRD.Kind }}Spec struct {
 	{{- range $fieldName, $field := .CRD.SpecFields }}
+	{{ if $field.IsRequired }} // +kubebuilder:validation:Required {{ end }}
```

Comment:
nit: I think using a `{{- if ...}}` instead of `{{ if ...}}` will prevent the generator from printing newlines if `field.IsRequired` is `false`

Reply:
Thanks for the pointer :)